### PR TITLE
Update the caption for GitHubTeacher meme

### PR DESCRIPTION
**Pull request by githubteacher**
@allthedoll does this look good? Do you think I'm funny?

### CODE REVIEW

**Review by allthedoll — CHANGES_REQUESTED** [review]
You talk about crytocurrency far too much, friend. Can't we go back to a simpler time when we talked about tacos?

**Comment by githubteacher** [comment]
General feedback belongs here

File: _posts/0000-01-14-githubteacher.md [hunk 1]
```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![fry](https://cloud.githubusercontent.com/assets/16547949/25400891/fb18941a-29c1-11e7-82b9-35e589443a60.jpg)
 {: .center}
 
-Caption here
+Not sure if Bitcoin is going up or down.
```
Comment (githubteacher) [inline _posts/0000-01-14-githubteacher.md]: This is a specific line comment

File: _posts/0000-01-14-githubteacher.md [hunk 2]
```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![fry](https://cloud.githubusercontent.com/assets/16547949/25400891/fb18941a-29c1-11e7-82b9-35e589443a60.jpg)
```
Comment (githubteacher) [inline _posts/0000-01-14-githubteacher.md:6]: Multiple comments

**Review by githubteacher — COMMENTED** [review]
👍

**Review by allthedoll — APPROVED** [review]
This is us:

us

**Comment by davidschaefer** [comment]
Now *that's* what I'm taco'n 'bout!

**Pull request merged**